test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/HistoryTests.cs

```diff
@@ -83,6 +83,34 @@ public async Task GivenNoType_WhenGettingSystemHistory_TheServerShouldReturnTheA
             Assert.NotEmpty(readResponse.Resource.Entry);
         }
 
+        [Fact]
+        public async Task GivenAType_WhenGettingResourceHistory_TheServerShouldReturnTheAppropriateBundleSuccessfullyWithResponseStatus()
```

Comment:
>GivenAType_WhenGettingResourceHistory_TheServerShouldReturnTheAppropriateBundleSuccessfullyWithResponseStatus [](http://example.com/codeflow?start=26&length=109)

>[http://hl7.org/fhir/bundle.html](http://hl7.org/fhir/bundle.html)

response: Results of execution (transaction/batch/history)

It's good to have history test, but can we also have test for transaction and batch actions?

You can find them by `BatchTest` and `TransactionTest`

Reply:
Looks like we are already covered for Batch and Transaction